### PR TITLE
Updating Saved Dashboard Feature with Backend  

### PR DESCRIPTION
**Pull request by evc29**
This code is implementing my feature for M4, updating the Saved Restaurants Dashboard with the backend code. This adds a Visited Restaurants table in the backend and adding endpoints so that we can dynamically add and remove from both Liked Restaurants and Visited Restaurants table in the backend. This is used in my Saved Restaurants Dashboard by fetching liked restaurant data then moving the restaurants into visited or removing all together. Adding reviewers for good Github practices!

### CODE REVIEW

File: team/front-end/.DS_Store
Comment (jadatwo) [inline team/front-end/.DS_Store:1]: can delete later

**Review by jadatwo — APPROVED** [review]
Looks good, dynamically adds and removes correctly, will need to remove .DS_store file to avoid conflict. ready to merge!

**Review by aliyabedeen — APPROVED** [review]
Looks great, adds and removes restaurants from want to try to visited accurately. Ready to merge!

**Review by addie-p — APPROVED** [review]
The changes look good - pulled them and tested them in my local env and the saved restaurants feature works in conjunction with the server-side implementation of the Restaurants, Liked Restaurants, and Visited Restaurants table.

**Review by cding888 — APPROVED** [review]
looks good to push to m4-integrate-main

**Pull request merged**